package.json

```
@@ -5,6 +5,14 @@
   "main": "dist/lexer.cjs",
   "module": "dist/lexer.js",
   "types": "types/lexer.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/lexer.js",
+      "require": "./dist/lexer.cjs"
+    },
+    "./package.json": "./package.json",
+    "./": "./"
```

```suggestion
    ".": {
      "module" "./dist/lexer.js",
      "import": "./dist/lexer.js",
      "require": "./dist/lexer.cjs"
    }
```

I've personally come around on the idea of using "module" for bundlers here.

Also I prefer not to expose unnecessary files. I understand there were issues with package.json loading but a custom node_modules lookup and package.json loader is 5 lines of a while loop so I don't think we should bend the entire JS ecosystem to saving these lines for metadata approaches.

---

Good point! Agree, that it's pretty trivial to do a custom lookup function. Just committed your suggestions.